relay-filter/src/localhost.rs

```diff
@@ -34,29 +32,22 @@ pub fn matches(event: &Event) -> bool {
 }
 
 /// Filters events originating from the local host.
-pub fn should_filter(event: &Event, config: &FilterConfig) -> Result<(), FilterStatKey> {
+pub fn should_filter<F>(item: &F, config: &FilterConfig) -> Result<(), FilterStatKey>
+where
+    F: Filterable,
+{
```

Comment:
nit: should we inline the `Filterable` impl, since there's only one?
```suggestion
pub fn should_filter<F: Filterable>(item: &F, config: &FilterConfig) -> Result<(), FilterStatKey> {
```

Reply:
👍 I inlined it everywhere now, except where it would break the signature into multiple lines.